NavigationAndTabBarProject/NavigationAndTabBarProject/TabBar/TabBarController.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  TabBarController.swift
+//  NavigationAndTabBarProject
+//
+//  Created by Rishat on 28.09.2020.
+//  Copyright © 2020 Rishat Latypov. All rights reserved.
+//
+
+import UIKit
+
+class TabBarController: UITabBarController {
+    
+    
+    
+    public override func viewDidLoad() {
+        
+           super.viewDidLoad()
+           }
+}
```

Comment:
Для чего этот класс?
```suggestion
class TabBarController: UITabBarController {
    public override func viewDidLoad() {
        super.viewDidLoad()
    }
}
```